### PR TITLE
Fixes #10365: Enable gzip compression on text resources

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/10365

### CODE REVIEW

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
Commit modified

**Comment by amousset** [comment]
OK, merging this PR

File: rudder-web/src/main/resources/rudder-apache-common.conf [hunk 1]
```diff
@@ -21,6 +21,16 @@ ErrorDocument 503 /rudder-loading.html
...
 
+# We do not require mod_deflate as it is not strictly necessary
+<IfModule mod_deflate.c>
+<IfModule mod_filter.c>
+  # Enable gzip encoding on textual resources
+  AddOutputFilterByType DEFLATE text/plain text/html text/xml text/javascript text/css
+  AddOutputFilterByType DEFLATE application/xml application/xhtml+xml application/rss+xml image/svg+xml
+  AddOutputFilterByType DEFLATE application/json application/javascript application/x-javascript
+</IfModule>
+/IfModule
```
Comment (ncharles) [inline rudder-web/src/main/resources/rudder-apache-common.conf]: missing < >

**Pull request merged**